website/app/modifiers/doc-track-event.js

```diff
@@ -9,14 +9,10 @@ import { inject as service } from '@ember/service';
 import { assert } from '@ember/debug';
 
 export default class DocTrackEvent extends Modifier {
-  @service fastboot;
+  @service eventTracking;
 
   modify(element, _positional, named) {
-    if (
-      // Only attempt to do something if we are in the right environment
-      this.fastboot.isFastBoot ||
-      window.fathom == null
-    ) {
+    if (!this.eventTracking.isEnabled) {
```

Comment:
Looking at this with fresher eyes I'm not sure this entire `if` block is actually necessary as the same check now happens [here](https://github.com/hashicorp/design-system/pull/2167/files#diff-b5adddb0d69ebd2ee22d67a269141428dda6e318176c851ef8158f9fd700cbeeR18) as well

Reply:
It's not strictly necessary, but it prevents the event listener from being added unnecessarily

Reply:
+1 on @zamoore approach